pkg/coreos/actuator_reconcile.go

```diff
@@ -97,6 +97,31 @@ func (c *actuator) cloudConfigFromOperatingSystemConfig(ctx context.Context, con
 			})
 		}
 
+		if unit.Name == "containerd.service" && config.Spec.EnableContainerD {
```

Comment:
Can we add a constant for this service name like for the kubelet: https://github.com/gardener/gardener/blob/master/pkg/apis/core/v1beta1/constants/types_constants.go#L238-L239 and use it?

Reply:
Done